### PR TITLE
Nv 332 get user environments api

### PR DESCRIPTION
**Pull request by ninjadev101**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
#481

### CODE REVIEW

File: apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.usecase.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Injectable, Scope } from '@nestjs/common';
+import { EnvironmentEntity, EnvironmentRepository } from '@novu/dal';
+import { GetMyEnvironmentsCommand } from './get-my-environments.command';
+
+@Injectable({
+  scope: Scope.REQUEST,
```
Comment (davidsoderberg) [inline apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.usecase.ts:6]: A question, is it needed to set scope here?

**Comment by davidsoderberg** [comment]
Great work @ninjadev101 🎉

**Review by davidsoderberg — APPROVED** [review]

File: apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.command.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { CommandHelper } from '../../../shared/commands/command.helper';
+import { OrganizationCommand } from '../../../shared/commands/organization.command';
+
+export class GetMyEnvironmentsCommand extends OrganizationCommand {
```
Comment (scopsy) [inline apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.command.ts:4]: Nice for noticing the OrganizationCommand!

**Review by scopsy — APPROVED** [review]
Looks awesome!

File: apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.usecase.ts (hunk 1 above)
Comment (scopsy) [reply apps/api/src/app/environments/usecases/get-my-environments/get-my-environments.usecase.ts:6]: Looks like not, but it's not making a harm either I think.

**Pull request merged**